### PR TITLE
#161 Add YAML front matter to existing ideas 

### LINKED ISSUE #161: Create yaml front matter for all committed ideas
Add [yaml front matter](https://jekyllrb.com/docs/frontmatter/) like e.g. https://raw.githubusercontent.com/status-im/ideas/master/ideas/121-swarm-compensation/README.md

```
---
id: 121-swarm-compensation
title: Swarm Compensation Experiments
status: in-progress
created: 2018-04-02
contributors:
    - martinklepsch
    - oskarth
---
```

to all committed ideas. This can be used to programmatically deal with ideas.

### PR DESCRIPTION
**Pull request by alexkohler**
- I wasn't completely sure on `category` for these (I think I listed them all as core), please let me know if any of them should be changed. 
- It appears `clear-roles` is yes for every idea (except the template), so I opted to make these all say yes, however, I'm a little unclear on the definition.

Fixes #161.

### CODE REVIEW

File: ideas/123-whisper-visualization.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+---
+id: 123-whisper-visualization
+title: Whisper Visualization
+status: Draft
+created: 2018-04-06
+category: core
```
Comment (oskarth) [inline ideas/123-whisper-visualization.md]: research probably, not related to any core okrs

File: ideas/123-whisper-visualization.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+lead-contributor: divan
+contributors:
+    - divan
+exit-criteria: no
+success-metrics: no
+clear-roles: yes
```
Comment (oskarth) [inline ideas/123-whisper-visualization.md]: not clear roles because there's only one contributor here

File: ideas/117-message-ordering.md [hunk 3]
```diff
@@ -1,3 +1,19 @@
+---
```
Comment (oskarth) [inline ideas/117-message-ordering.md]: can unstage this, I believe @cammellos wanted this idea removed

File: ideas/145-identity.md [hunk 4]
```diff
@@ -1,3 +1,18 @@
+---
+id: 145-identity
+title: Self-soverign Identity
+status: Draft
+created: 2018-05-04
+category: core
+lead-contributor: divan
```
Comment (oskarth) [inline ideas/145-identity.md]: wrong lead contributor and contributors here, looks like copy paste gone wrong

File: ideas/150-gas-abstraction.md [hunk 5]
```diff
@@ -1,3 +1,21 @@
+---
+id: 150-gas-abstraction
+title: Gas Abstraction
+status: Draft
+created: 2018-02-01
+category: core
+lead-contributor: Ricardo Guilherme Schmidt
+contributors:
+    - Ricardo Guilherme Schmidt
```
Comment (oskarth) [inline ideas/150-gas-abstraction.md]: Would you mind syncing these to contributors.yaml file / add GH names if it makes sense? E.g. Ricardo should be @3esmit, Richard @richard-ramos @iurimatias, Alex (from Mist IIRC) @alexvandesande.

File: ideas/152-friends-recovery.md [hunk 6]
```diff
@@ -1,3 +1,19 @@
...
+status: Draft
+created: 2018-04-07
+category: core
+lead-contributor: Ricardo Guilherme Schmidt
+contributors:
+    - Ricardo Guilherme Schmidt
+    - Richard Ramos
+exit-criteria: yes
+success-metrics: yes
+clear-roles: yes
```
Comment (oskarth) [inline ideas/152-friends-recovery.md]: not clear role here I believe

File: ideas/61-app-structure-refinement.md [hunk 7]
```diff
@@ -1,3 +1,24 @@
+---
+id: 061-structure refinement
+title: App structure refinement
+status: Active
```
Comment (oskarth) [inline ideas/61-app-structure-refinement.md]: think this is Completed

File: ideas/94-wallet-compatibility.md [hunk 8]
```diff
@@ -1,3 +1,19 @@
...
+status: Draft
+created: 2018-03-20
+category: core
+lead-contributor: pilu
+contributors:
+    - pilu
+    - asemiankevich
+exit-criteria: yes
+success-metrics: no
+clear-roles: yes
```
Comment (oskarth) [inline ideas/94-wallet-compatibility.md]: not clear roles, lacking PM/UX(R)

File: ideas/150-gas-abstraction.md (hunk 5 above)
Comment (alexkohler) [reply ideas/150-gas-abstraction.md]: Sure - what about the handles for Hester Bruikman, Denis Sharypin, and Rachel Hamlin?

**Comment by alexkohler** [comment]
@oskarth - I've addressed all of your comments - I still have one open question - what's Denis Sharypin's Github username?

**Comment by oskarth** [comment]
Thank you @alexkohler! Unfortunately https://github.com/status-im/ideas/pull/230 resulted in some merge conflict. In hindsight I should've merged them the other way around probably. Would you mind resolving and we can merge this one?

**Comment by alexkohler** [comment]
@oskarth No worries! :smile: I've merged master.

**Pull request merged**